Gltf/GltfGenerator/GltfGenerator.csproj

```
@@ -1,22 +1,32 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <LangVersion>preview</LangVersion>
-    <Version>2.0.0</Version>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
   </PropertyGroup>
+  
+  <!-- Package references needed for source generators -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.9.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" PrivateAssets="all" />
+  </ItemGroup>
+
+  <!-- External dependencies -->
+  <!-- See https://github.com/dotnet/roslyn-sdk/tree/main/samples/CSharp/SourceGenerators -->
+  <ItemGroup>
+    <PackageReference Include="System.CodeDom" Version="5.0.0" GeneratePathProperty="true" PrivateAssets="all" />
```

```suggestion
    <PackageReference Include="System.CodeDom" Version="5.0.0" PrivateAssets="all" />
```
I'll comment on why the removal of this further down...